n8n-db-provisioner.sh

```
@@ -0,0 +1,257 @@
+    exec_sqlite "PRAGMA table_info(tag_entity);" 2>/dev/null || echo "No tag_entity table"
+}
+
+# Step 3: Create user isolation structure
+create_user_isolation() {
+    local user_id="$1"
+    local user_email="$2"
+    
+    log_info "Creating isolation structure for user: $user_id ($user_email)"
+    
+    # Check if project table exists
+    local has_projects=$(exec_sqlite "SELECT count(*) FROM sqlite_master WHERE type='table' AND name='project_entity';")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)